From 954ee437d3f1dc2649e1a1b201fd7a69b3fdc949 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Fri, 3 Jul 2015 14:03:53 +0800
Subject: [PATCH 0157/5242] MLK-11123-20 ARM: dts: disable otg features for
 imx6q-arm2 board

commit  85537c73cd3d5020cfdb174c9a5c30e690981348 from
https://source.codeaurora.org/external/imx/linux-imx.git

Disable all otg features for otg1 port of imx6q-arm2 board, then
the otg port can only do role swtich by ID pin state.

Signed-off-by: Li Jun <jun.li@freescale.com>
(cherry picked from commit 70c4fe93ac7064f5128ae2275b766fb00c4cd6ef)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6q-arm2.dts |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/imx6q-arm2.dts b/arch/arm/boot/dts/imx6q-arm2.dts
index 953a5b5..e8e5016 100644
--- a/arch/arm/boot/dts/imx6q-arm2.dts
+++ b/arch/arm/boot/dts/imx6q-arm2.dts
@@ -194,6 +194,9 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usbotg>;
 	disable-over-current;
+	srp-disable;
+	hnp-disable;
+	adp-disable;
 	status = "okay";
 };
 
-- 
1.7.9.5

